scalableminds/webknossos
# Rename Open Tasks to Pending

- Rephrased "open" tasks as "pending" to improve clarity.
- Slightly redesign the renamed the "Available Task Assignment" list view to improve clarity and ease-of-use.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Navigate to https://webknossos.brain.mpg.de/reports/openTasks and check wording etc

### Issues:
- fixes #7012
- fixes #7011

------
(Please delete unneeded items, merge only when none are left open)
